docs/services/azure-services-guide.md

```diff
@@ -13,11 +10,10 @@ Crossplane managed resources and the official Wordpress Docker image.
 
 1. [Pre-requisites](#pre-requisites)
 2. [Preparation](#preparation)
-3. [Set Up AKS Cluster](#set-up-aks-cluster)
-4. [Set Up Crossplane](#set-up-crossplane)
-5. [Install Wordpress](#install-wordpress)
-6. [Uninstall](#uninstall)
-7. [Conclusion and Next Steps](#conclusion-and-next-steps)
+3. [Set Up Crossplane](#set-up-crossplane)
```

Comment:
I like the simpler ToC, and shorter doc in general, it's a big improvement!

Suggest the following ToC:

1. Pre-requisites
1. Preparation
1. Set Up Crossplane
    1. Install Crossplane
    1. Cloud Provider
    1. Resource Classes
    1. Configure Managed Service Access (1)
1. Provision MySQL
   1. Resource Claim
   1. VNet Rule
1. Install Wordpress
1. Clean Up
1. Conclusion and Next Steps

(1) suggest moving the "Virtual Network with the Microsoft.Sql service endpoint enabled" into a very short section here (Configure Managed Service Access), so that step can be common across the guides.

I used a similar pattern in the manual steps section of these posts:
 - https://blog.crossplane.io/gcp-secure-connectivity-for-gke-cloudsql/
 - https://blog.crossplane.io/aws-secure-connectivity-for-eks-rds/
 - https://blog.crossplane.io/azure-secure-connectivity-for-aks-azure-db/